tests/e2e/common-operator-integ-suite.sh

```diff
@@ -241,9 +241,12 @@ if [ "${SKIP_BUILD}" == "false" ]; then
     BUNDLE_IMG="${BUNDLE_IMG}" \
     OPENSHIFT_PLATFORM=false \
     make bundle bundle-build bundle-push
-     
+
     # Install OLM in the cluster because it's not available by default in kind.
-    ${OPERATOR_SDK} olm install
+    if [ "${OLM_VERSION}" != "" ]; then
+      OLM_INSTALL_ARGS="--version ${OLM_VERSION}"
+    fi
+    ${OPERATOR_SDK} olm install "${OLM_INSTALL_ARGS}"
```

Comment:
Isn't this failing when OLM_INSTALL_ARGS is undefined?

Reply:
It will fail if undefined as the script is using `set -eu -o pipefail`

Reply:
good point. fixed